src/Project.php

```diff
@@ -2669,6 +2669,200 @@ public function showDebug()
+        if (
+            empty(array_filter($projects, function ($project) {
+                $state = ProjectState::getById($project->fields['projectstates_id']);
+                if ($state !== false) {
+                    if ($state->fields['is_finished']) {
+                        return false;
+                    }
+                }
+
+                return true;
+            }))
+        ) {
+            return;
+        }
+
+        $options = [
+            'criteria' => [
+                [
+                    'link' => 'AND',
+                    'field' => 87,
+                    'searchtype' => 'equals',
+                    'value' => 'myself'
+                ],
+                [
+                    'link' => 'OR',
+                    'field' => 24,
+                    'searchtype' => 'equals',
+                    'value' => 'myself'
+                ]
+            ]
+        ];
+
+        $displayed_row_count = min(count($projects), 5);
+        $total_row_count = count($projects);
+        $search_url = self::getSearchURL() . '?' . Toolbox::append_params($options);
+        $title = Html::makeTitle(__('Your projects'), $displayed_row_count, $total_row_count);
+        $main_header = "<a href='$search_url'>$title</a>";
+
+        $twig_params = [
+            'class'       => 'table table-borderless table-striped table-hover card-table',
+            'header_rows' => [
+                [
+                    [
+                        'colspan'   => 4,
+                        'content'   => $main_header
+                    ]
+                ],
+                [
+                    [
+                        'content' => __('Name'),
+                        'style'   => 'width: 30%'
+                    ],
+                    [
+                        'content' => _n('State', 'States', 1),
+                        'style'   => 'width: 30%'
+                    ],
+                    [
+                        'content' => __('Priority'),
+                        'style'   => 'width: 30%'
+                    ],
+                    [
+                        'content' => __('Percent done'),
+                        'style'   => 'width: 10%'
+                    ]
+                ]
+            ],
+            'rows' => []
+        ];
+
+        foreach ($projects as $key => $project) {
+            if ($key >= $displayed_row_count) {
+                break;
+            }
+
+            $name = $project->getLink();
+            $priority = CommonITILObject::getPriorityName($project->fields['priority']);
+            $percent_done = $project->fields['percent_done'] . '%';
+
+            $state = ProjectState::getById($project->fields['projectstates_id']);
+            if ($state !== false) {
+                $state_cell = '<div class="priority_block" style="border-color: ' . $state->fields['color'] . '"><span class="me-1" style="background: ' . $state->fields['color'] . '"></span>' . $state->fields['name'];
```

Comment:
I am not sure about using a `priority_xxxx` class for something that isn't a priority.
Maybe we could create a more generic alias for this css class ?